site/app/views/admin/UsersView.php

```diff
@@ -34,6 +35,7 @@ public function listStudents($sorted_students, $reg_sections, $rot_sections, $do
             "delete_user_url" => $this->core->buildCourseUrl(['delete_user']),
             "return_url_upload_class_list" => $this->core->buildCourseUrl(['users', 'upload']) . '?' . http_build_query(['list_type' => 'classlist']),
             'view_grades_url' => $this->core->buildCourseUrl() . '/users/view_grades',
+            'view_latedays_url' => $this->core->buildCourseUrl() . '/users/view_latedays',
```

Comment:
It's usually preferred to do `$this->core->buildCourseUrl(['users', 'view_latedays'])` instead